MANIFEST.in

```diff
@@ -32,7 +32,7 @@ global-exclude __pycache__  *.pyc
 include airflow/alembic.ini
 include airflow/api_connexion/openapi/v1.yaml
 include airflow/git_version
-include airflow/provider.yaml.schema.json
+include airflow/provider.runtime.yaml.schema.json
```

Comment:
I thought about it but I'd rather keep the name 'provider.yaml.schema.json` because all the IDE tools (and people matching them) would rather use provider.yaml.json.schema to edit provider.yaml. And in fact provider.yaml is already development-only. There are no provider.yaml files beside the development.

But  this thought leads me to another, better naming. The `provider.runtime.yaml.schama.json' shoudl be renamed `provider_info.schema.json` . This would be 1-1 with the entry-point name and there should be no yaml, because there is no yaml whatsoever n provider_info entrypoint.

Reply:
Yeah. Just pushed it. This is much better - and there is not a single mention of provider.yaml in the user-facing documentation. Also airflow package does not contain provider.yaml schema nor provider.yaml files, so this leaves provider.yaml and provider.yaml.schema.json as development-only-things, as it should be.